src/Integration.UnitTests/Helpers/SonarLintOutputTests.cs

```diff
@@ -41,15 +42,13 @@ public void Write_OutputsToWindow()
         {
             // Arrange
             var windowMock = new ConfigurableVsOutputWindow();
+            var serviceProviderMock = CreateConfiguredServiceProvider(windowMock);
```

Comment:
I did some minor cleanup on the existing tests to make them more similar to our current style i.e. helper methods and a variable called `testSubject`.

There is one new test below.